includes/frontend.php

```diff
@@ -1408,6 +1408,7 @@ protected function get_init_settings() {
 				'breakpoints' => Plugin::$instance->breakpoints->get_breakpoints_config(),
 			],
 			'version' => ELEMENTOR_VERSION,
+			'version_pro' => defined( 'ELEMENTOR_PRO_VERSION' ) ? ELEMENTOR_PRO_VERSION : '',
```

Comment:
I don't think we had any way to read the Pro version from the Core version with JS. Or is there another way to do it?

Reply:
I could find any other JS way to do this.